src/SFA.DAS.Roatp.CourseManagement.Web/Controllers/AddProviderContact/ProviderContactUpdateStandardsController.cs

```diff
@@ -49,6 +49,11 @@ public IActionResult PostUpdateStandardsEmailAndPhone(int ukprn, ProviderContact
         providerContactModel.UpdateExistingStandards = submitViewModel.UpdateExistingStandards;
         _sessionService.Set(providerContactModel);
 
+        if (submitViewModel.UpdateExistingStandards == true)
```

Comment:
If you rename this in proper question format HasOptedToUpdateExistingStandards then you can remove the redundant `== true` and it will read a lot better.

Reply:
not redundant because it is am nullable value (a user may not select either). but updated to the clearer 'is true'